docs/050-breaking-changes.rst

```diff
@@ -272,6 +273,83 @@ Syntax
   ``override``, ``partial``, ``promise``, ``reference``, ``sealed``,
   ``sizeof``, ``supports``, ``typedef`` and ``unchecked``.
 
+.. _interoperability:
+
+Interoperability with older Contracts
+=====================================
+
+It is still possible to interface with contracts written for Solidity versions prior to
+v0.5.0 by defining interfaces for them. Consider you have the following pre-0.5.0
+contract already deployed:
+
+::
+
+   // This will not compile
```

Comment:
This testing directive is confusing for users - does it also work to add "This will not compile with the current version of the compiler"?

Reply:
I'm actually not sure - I'll try! If not we may want to change that.